pkgs/applications/radio/yaac/default.nix

```diff
@@ -0,0 +1,48 @@
+{ ant, fetchsvn, jdk, jre8, lib, makeWrapper, stdenv, unzip }:
+
+stdenv.mkDerivation rec {
+
+  pname = "yaac";
+  version = "1.0-beta182";
```

Comment:
```suggestion
  version = "229";
```
Would that be more correct? I am not sure

Reply:
I don't believe so, the versioning of the application is different from the sourceforge revision.

![image](https://github.com/NixOS/nixpkgs/assets/11077277/33b0678c-4dfa-4108-b594-8bafbb29502d)